src/explore-education-statistics-common/src/modules/release/components/ReleaseDataFilesAccordion.tsx

```diff
@@ -0,0 +1,203 @@
+import AccordionSection from '@common/components/AccordionSection';
+import Details from '@common/components/Details';
+import SectionBreak from '@common/components/SectionBreak';
+import Accordion from '@common/components/Accordion';
+import { Release } from '@common/services/publicationService';
+import { FileInfo } from '@common/services/types/file';
+import classNames from 'classnames';
+import orderBy from 'lodash/orderBy';
+import React, { ReactNode } from 'react';
+import styles from './ReleaseDataFilesAccordion.module.scss';
+
+interface Props {
+  release: Release;
+  renderAllFilesButton?: ReactNode;
+  renderCreateTablesButton: ReactNode;
+  renderDataCatalogueLink: ReactNode;
+  renderDataGuidanceLink: ReactNode;
+  renderDownloadLink: (file: FileInfo) => ReactNode;
+  renderRelatedDashboards?: ReactNode;
+  onSectionOpen?: (accordionSection: { id: string; title: string }) => void;
+  showDownloadFilesList?: boolean;
+}
+
+const ReleaseDataFilesAccordion = ({
+  release,
+  renderAllFilesButton,
+  renderCreateTablesButton,
+  renderDataCatalogueLink,
+  renderDataGuidanceLink,
+  renderDownloadLink,
+  renderRelatedDashboards,
+  onSectionOpen,
+  showDownloadFilesList = false,
+}: Props) => {
+  const dataFiles = orderBy(
+    release.downloadFiles.filter(file => file.type === 'Data'),
+    ['name'],
+  );
+
+  const ancillaryFiles = orderBy(
+    release.downloadFiles.filter(
+      file => file.type === 'Ancillary' && file.name !== 'All files',
+    ),
+    ['name'],
+  );
+
+  const hasAllFilesButton =
+    (dataFiles.length > 0 || ancillaryFiles.length > 0) && renderAllFilesButton;
+
+  return (
+    <div className={styles.container}>
+      <Accordion
+        id="data-accordion"
+        showOpenAll={false}
+        onSectionOpen={onSectionOpen}
+      >
+        {release.downloadFiles && (
+          <AccordionSection
+            id="explore-data-and-files-section"
```

Comment:
Not a big deal, but we don't really need to postfix either of these sections with 'section'